app/views/admin/organisations_about/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<% content_for :page_title, @organisation.name %>
+<% content_for :title, @organisation.name %>
+<% content_for :context, organisation_context_block(current_user, @organisation)  %>
+<% content_for :title_margin_bottom, 4 %>
+
+<p class="govuk-body">
+  <%= view_on_website_link_for @organisation, class: "govuk-link", target: "blank" %>
+</p>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= render "components/secondary_navigation", {
```

Comment:
The `secondary_navigation` component needs to be within a full-width container - in this case the tabs overflow the container.

Reply:
great catch have sorted thanks